subsys/bluetooth/audio/aics.c

```diff
@@ -67,34 +67,37 @@ static ssize_t read_description(struct bt_conn *conn,
 	BT_GATT_SECONDARY_SERVICE(BT_UUID_AICS),                               \
 	BT_GATT_CHARACTERISTIC(BT_UUID_AICS_STATE,                             \
 			       BT_GATT_CHRC_READ | BT_GATT_CHRC_NOTIFY,        \
-			       BT_GATT_PERM_READ_ENCRYPT,                      \
+			       BT_GATT_PERM_READ_ENCRYPT | BT_GATT_PERM_READ_128B_ENTROPY,   \
```

Comment:
```suggestion
			       BT_GATT_PERM_READ_128B_ENTROPY,   \
```

Applies multiple places since we don't need ENCRYPT and 128B_ENTROPY